ipi/utils/io/backends/io_xyz.py

```diff
@@ -69,65 +68,70 @@ def print_xyz(atoms, cell, filedesc=sys.stdout, title=""):
         filedesc.write("%8s %12.5e %12.5e %12.5e\n" % (lab[i], qs[3*i], qs[3*i+1], qs[3*i+2]))
 
 
+# Cell type patterns
+cell_re = [re.compile('# CELL.abcABC.: ([-0-9.Ee ]*) '), re.compile('# CELL.GENH.: ([-0-9.Ee ]*)'),
```

Comment:
here I am guilty. partially too lazy to type \(, partially wanted to leave some leeway if someone writes CELL[abcABC]. harmless IMO